File: level_1/최대공약수와 최소공배수.java
class Solution {
    public int[] solution(int n, int m) {
        int min = Math.min(n, m);
        int i;
        for (i = min; i > 0; i--)
            if (n % i == 0 && m % i == 0)
                break;

        int lcm = n * m / i;

        int[] answer = { i, lcm };
        return answer;
    }
}

File: level_1/최대공약수와 최소공배수.js
const solution = (n, m) => [gcd(n, m), lcm(n, m, gcd(n, m))];

const gcd = (n, m) => {
  if (m % n != 0) return gcd(m % n, n);
  return n;
};

const lcm = (n, m, gcd) => {
  return (n * m) / gcd;
};
